packages/react-ui-ag/src/Modals/EmailModal.js

```diff
@@ -0,0 +1,77 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import classnames from 'classnames'
+import autobind from 'autobind-decorator'
+import { ClosableModal, Form, Field, Button } from '@rentpath/react-ui-core'
+
+@themed(/^EmailModal/, { pure: true })
+export default class EmailModal extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    header: PropTypes.node,
+    label: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+    footer: PropTypes.node,
+    onSubmit: PropTypes.func,
+    error: PropTypes.node,
+    email: PropTypes.string,
+    field: PropTypes.object,
+    submitButton: PropTypes.object,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  @autobind
+  onSubmit(event, fields) {
+    const { onSubmit } = this.props
+
+    if (onSubmit) onSubmit(event, fields)
+  }
```

Comment:
Is there any need to have `this.onSubmit` if all it does is turnaround and call the `this.props.onSubmit` with the same arguments it receives? Why not just use `onSubmit={this.props.onSubmit}` down in the render function?

Reply:
Good point, I expected I'd need to add something and never did.

Reply:
Fixed.

Reply:
Also my preference is to name all handlers starting with "handle", so `handleSubmit` in this case unless you decide to remove this.